### DISCUSSION TITLE
Having an issue with importing the library inside next js

### DISCUSSION THREAD

**Discussion by AnanduP03** [roerohan/react-vnc#50] [Q&A]
I have an issue with importing the react-vnc package inside my project. I'm using Next JS 13 and typescript in my project. When I'm importing using the way mentioned in the readme it's showing this error

Could not find a declaration file for module 'react-vnc'
{app/node
modules/ pnpm/react-vnc@1.0.0_react
dom@18.2.0 react-scripts@5.0.1 react@18.2.0/node modules/react-vnc/dist/index.es.js' implicitly has an 'any' type.

I went through the packages folder and found the types folder inside the dist folder but my project is returning this error.

Can someone suggest what I should do to fix this.

**Comment by roerohan** [comment]
```
app/node_modules/pnpm/react-vnc@1.0.0_react
```

Why is the `react-vnc` library installed inside a `pnpm` directory?

Could you show me the dependencies section in your `package.json`, and the `node_modules/react-vnc/package.json` file?
v1.0.0 has been out for a while and I haven't gotten any complaints, so I think it might be an issue specific to your environment.

**Comment by AnanduP03** [comment]
i use pnpm package manager instead of npm so i used 
```pnpm add react-vnc```

package.json
![image](https://github.com/roerohan/react-vnc/assets/104244609/470f45d3-c79d-44bb-bc84-556079b91e0b)

node_modules/react-vnc/package.json
![image](https://github.com/roerohan/react-vnc/assets/104244609/a810f139-ff03-47d0-9b0d-0aaa7f72a512)

**Reply by AnanduP03** [reply]
any updates???
